tech-far-hub/src/components/site-layout.tsx

```diff
@@ -66,7 +66,7 @@ const SiteLayout = ({ children, pagePath, breadCrumbs, className }: ILayoutProps
         src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=Executive%20Office%20of%20the%20President&subagency=USDS"
         id="_fed_an_ua_tag"
       ></script>
-      <script src="https://touchpoints.app.cloud.gov/touchpoints/e93bb39d.js"></script>
+      <Script src="https://touchpoints.app.cloud.gov/touchpoints/e93bb39d.js"></Script>
```

Comment:
Does this script load on every page?

Reply:
Yup, but it needs to --- we use touchpoints in two days.